media/stylus/components/wiki/page-buttons.styl

```diff
@@ -34,6 +34,15 @@
     }
 }
 
+@media $media-query-small-desktop {
+
+    .page-buttons, #page-buttons {
+        &.fixed {
+            position: relative;
+        }
+    }
+}
+
```

Comment:
Another option to consider is nesting this media query under the existing occurrence of the selector:

https://github.com/mozilla/kuma/pull/3137/files#diff-e2b6c4642cf93c8eab58fc0ecb41f3adR7